README.md

```diff
@@ -31,6 +31,59 @@ LEMMY_BB_BACKEND=https://example.com cargo run
 
 Then open [127.0.0.1:1244](http://127.0.0.1:1244) in your browser. You can login with your existing account.
 
+### Standalone
+
+Follow these steps to install only the lemmyBB frontend on your server.
```

Comment:
Should specify that this requires an existing Lemmy installation, and probably link to the install instructions as well. Also not sure if this works with Docker installation or native Lemmy installation, or both.

Reply:
I would put a title for this section like "Installation for existing Lemmy instance".